system/netdata.service.in

```diff
@@ -10,15 +10,15 @@ Type=simple
 User=netdata
 Group=netdata
 RuntimeDirectory=netdata
+CacheDirectory=netdata
+StateDirectory=netdata
+LogsDirectory=netdata
```

Comment:
My only concern with this change is that it requires SystemD 235+ (_see [NEWS](https://github.com/systemd/systemd/blob/54dee8541ba34f7613504d69bb1bbd8eabc422ba/NEWS#L1089-L1339)_) -- This _may_ break some users of NetData on older systems (_I'll have to check..._)

Reply:
Just correcting the link to [NEWS](https://github.com/systemd/systemd/blob/54dee8541ba34f7613504d69bb1bbd8eabc422ba/NEWS#L2636) file.

Not sure if there is an easy way to support new and old versions.

Reply:
Solution could be that we ship two service files? And let distribution decide which one to use. Because systemd 235+ definitely makes it neat.

Reply:
In theory I have no issues with providing different SystemD unit files for older systems. This does make our CI/CD a bit more complicated however as we'll be the ones primarily maintaining the packaging (_at least we're building automation to deliver on this soon_).

cc @Ferroin What are you thoughts here?

Reply:
For time being CI/CD can use the current service file. So it wont break. And there can be a note for distribution to copy new service file instead of old, if possible. And then in future new file can be made the default.

Reply:
Systemd 235 is from October 2017 (as per [the date on the tag](https://github.com/systemd/systemd/releases/tag/v235)). The only systems we officially support that have an older version are CentOS 6 (not sure of exact version), CentOS 7 (not sure of exact version), Debian 8 (215), Debian 9 (232) and Ubuntu 16.04 (229). Everything else we support is new enough that they'll have a newer version of Systemd.

Given this, I say we use the 235+ ones as default, and add special case checks to the install to use the old ones when needed.

Reply:
That's potentially a large no. of systems. But yes I agree we can move forward with this and support two different SystemD unit files depending on the version of SystemD we find on the system.

Reply:
OK, if we're going the split file route, then we need two files.

@amishmm, can you update this to have the first commit (the one that uses systemd to create the directories) create a separate unit file? The second one can be applied both to the existing file and the new one, and we can handle the stuff in the installer to use the correct unit file.

Reply:
Done. Follow up [comment](https://github.com/netdata/netdata/pull/7790#issuecomment-576696004).